.github/workflows/ci.yml

```diff
@@ -43,23 +60,11 @@ jobs:
         uses: taiki-e/install-action@nextest
 
       - name: Setup just
-        uses: extractions/setup-just@v1
+        uses: extractions/setup-just@v2
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Run sccache-cache
-        uses: mozilla-actions/sccache-action@v0.0.3
-
-      - name: Cache Rust
-        uses: Swatinem/rust-cache@v2
```

Comment:
Does the removal of the caches affect the build time negatively?

Reply:
`Swatinem/rust-cache@v2` is automatically included in rust-lang-action. sscache's value is questionable at large, and specifically at PR time, I left it in docker builds though.